api/app/views/mno_enterprise/jpi/v1/admin/invoices/index.json.jbuilder

```diff
@@ -1,4 +1,6 @@
 json.invoices(@invoices) do |invoice|
   json.extract! invoice, :id, :price, :started_at, :ended_at, :created_at, :updated_at, :paid_at, :slug
-  json.organization invoice.organization, :id, :name
+  if invoice.organization
```

Comment:
Doesn't an invoice always has an organization?

Reply:
I'm not sure if my local DB was corrupted, but I had an invoice that was only atached to a user and this was failing. 

It seems like invoice associations in hub are polymorphic -- and no validations to assure it has an org: 
https://github.com/maestrano/maestrano-hub/blob/2.2/app/models/invoice.rb#L73